health/health_json.c

```diff
@@ -76,6 +82,17 @@ void health_alarm_entry2json_nolock(BUFFER *wb, ALARM_ENTRY *ae, RRDHOST *host)
                    , (ae->flags & HEALTH_ENTRY_FLAG_SILENCED)?"true":"false"
     );
 
+    if (likely(ae->info)) {
+        char *m=NULL;
+
+        while ( m = strstr (ae->info, "$family") ) {
```

Comment:
We had talked about this initially, and I will remember you, after the first family is found and we overwrite it, it is not necessary to return for the beginning of the string, we can continue overwriting from the place we changed the value.

Another important thing here, `family` is a static value after the chart to be created, so we do not need to change the alarm entries, we can change the alarm that gives input value for alarm entry.

Reply:
After to review next function I was thinking that probably you want to keep the initial value for the original setence, so my suggestion is to create a modified `info` and use it to create the alarm entry.

Reply:
Hi Thiago! OK, I've used a pos to force strstr to go after the last replacement. I've also added using helper variables on all occurrences of find_and_replace with https://github.com/netdata/netdata/pull/10961/commits/ff67cdd2ca7c6441887ffa25c5052bc6f5ff6629